src/com/palmergames/bukkit/towny/object/Nation.java

```diff
@@ -260,14 +267,7 @@ else if (this.spawn != null)
 
 	public List<Resident> getAssistants() {
 
-		List<Resident> assistants = new ArrayList<>();
-		
-		for (Town town: towns)
-		for (Resident assistant: town.getResidents()) {
-			if (assistant.hasNationRank("assistant"))
-				assistants.add(assistant);
-		}
-		return assistants;
+		return towns.stream().flatMap(town -> town.getResidents().stream()).filter(assistant -> assistant.hasNationRank("assistant")).collect(Collectors.toList());
```

Comment:
this could probably stream this.getResidents()

Reply:
Agreed.